tests/bluetooth/tester/src/btp_mesh.c

```diff
@@ -1114,15 +1123,6 @@ static const struct bt_mesh_model root_models_alt[] = {
 #if defined(CONFIG_BT_MESH_RPR_SRV)
 	BT_MESH_MODEL_RPR_SRV,
 #endif
-#if defined(CONFIG_BT_MESH_DFD_SRV)
```

Comment:
Why did you remove models from this list?
Adding them separately on the same element seems like misusing of `BT_MESH_ELEM`.
I'm surprised that such declaration works, since the macro was intended to declare model list for separate elements.

Reply:
I did not get that; models are moved into different elements, they are not on the same element anymore!

 if you are talking about the first parameter of the BT_MESH_ELEM macro being 0, the first element is the location code not the element index on the list of element. http://www.bluetooth.com/specifications/assigned-numbers section 2.4.2.1
 
 If not so, I might need a bit more clarification on this.

Reply:
Yep, you are right. My mistake.